project/README.md

```diff
@@ -14,6 +14,16 @@
 
 
 ## React
+[Jayesh Tambe](https://github.com/jtambe)
+
+**Major**
+
+* [Developed example code for DOM manipulation using react](https://github.com/facebook/react/pull/9683)
+
+**Minor**
+
+* [I had added documentation for setting up react environment on ubuntu machine](That link is no longer in the repo)
```

Comment:
Do you have your old pull request for it? Or if it is part of the wiki, it should be discoverable by going to the wiki history